packages/wonder-blocks-dropdown/components/single-select-menu.js

```diff
@@ -77,15 +77,9 @@ export default class SingleSelectMenu extends React.Component<Props, State> {
         };
     }
 
-    toggleMenu() {
-        this.setState((prevState) => ({
-            open: !prevState.open,
-        }));
-    }
-
-    handleClose() {
+    handleOpenChanged(open: boolean) {
         this.setState({
```

Comment:
I mentioned this in the other PR but you can do `this.setState({open});` in a few places, if you want.